app/components/card.tsx

```diff
@@ -32,21 +38,44 @@ const variantStyles: Record<Variant, string> = {
 }
 
 export const Card = React.forwardRef<HTMLDivElement, PropsWithChildren<Props>>(
-  function Card({ icon, title, variant = 'light', className, children }, ref) {
+  function Card(
+    {
+      icon,
+      title,
+      link,
+      variant = 'light',
+      transparantCards,
+      className,
+      children,
+    },
+    ref,
+  ) {
     const Icon = sbGradientIconMap[icon ?? '']
+
+    const baseClasses = 'flex flex-col gap-4 rounded-3xl border p-6'
+    const shadowCardClass = !transparantCards ? 'shadow-card' : ''
+    const backgroundClass = transparantCards
+      ? 'bg-transparent'
+      : variantStyles[variant]
+
+    const cardClasses = clsx(
+      baseClasses,
+      shadowCardClass,
+      backgroundClass,
+      className,
+    )
```

Comment:
You can use an object notation within clsx that will make this a bit easier to read: 

```suggestion
    const cardClasses = clsx('flex flex-col gap-4 rounded-3xl border p-6', 
      transparentCard ? 'bg-transparent' : variantStyles[variant],
      {
        'shadow-card': !transparentCards,
       }
    )
```

Now i would also move this inside of the `className={}` prop as there is not really a use to have it in a separate variable.